src/Kurrent.Client/PersistentSubscriptions/KurrentPersistentSubscriptionsClient.Read.cs

```diff
@@ -1,12 +1,67 @@
 using System.Threading.Channels;
 using EventStore.Client.PersistentSubscriptions;
 using EventStore.Client.Diagnostics;
+using EventStore.Client.Serialization;
 using Grpc.Core;
-
+using Kurrent.Client.Core.Serialization;
 using static EventStore.Client.PersistentSubscriptions.PersistentSubscriptions;
 using static EventStore.Client.PersistentSubscriptions.ReadResp.ContentOneofCase;
 
 namespace EventStore.Client {
+	public class SubscribeToPersistentSubscriptionOptions {
+		/// <summary>
+		/// The size of the buffer.
+		/// </summary>
+		public int BufferSize { get; set; } = 10;
+
+		/// <summary>
+		/// The optional user credentials to perform operation with.
+		/// </summary>
+		public UserCredentials? UserCredentials { get; set; } = null;
+
+		/// <summary>
+		/// Allows to customize or disable the automatic deserialization
+		/// </summary>
+		public OperationSerializationSettings? SerializationSettings { get; set; }
+	}
+
+	public class PersistentSubscriptionListener {
+#if NET48
+		/// <summary>
```

Comment:
The docs comment should not be inside pragma

Reply:
Fixed in https://github.com/EventStore/EventStore-Client-Dotnet/pull/329/commits/e8385b38d24f5eb138b8393a4acbcdc0c6416a9b